lib/Runtime/Library/ArrayBuffer.h

```diff
@@ -103,8 +128,12 @@ namespace Js
         static const uint32 MaxArrayBufferLength = 1 << 30;
 #endif
         virtual bool IsValidAsmJsBufferLength(uint length, bool forceCheck = false) { return false; }
-        virtual bool IsValidVirtualBufferLength(uint length) { return false; }
+        virtual bool IsArrayBuffer() override { return true; }
+        virtual bool IsSharedArrayBuffer() override { return false; }
+        virtual ArrayBuffer * GetAsArrayBuffer() override { return ArrayBuffer::FromVar((Var)this); }
```

Comment:
Elsewhere too

---
In reply to: [78116315](https://github.com/Microsoft/ChakraCore/pull/1533#discussion_r78116315) [](ancestors = 78116315)